src/components/Onboarding/OnboardingPage2.tsx

```diff
@@ -3,39 +3,45 @@ import { Image, Text, TextInput, View } from 'react-native';
 import { TouchableOpacity } from 'react-native-gesture-handler';
 import * as Progress from 'react-native-progress';
 import { KeyboardAwareScrollView } from 'react-native-keyboard-aware-scroll-view';
+import { useNavigation } from '@react-navigation/core';
 import styles from '../../styles/OnboardingStyles';
+import { CreateHomeInput } from '../../API';
 
 interface Props {
-  setPage(page: string): void;
   setHomeData({}): void;
-  homeData: Record<string, string>;
+  homeData: CreateHomeInput;
 }
 
-const Page2: React.FC<Props> = ({ setPage, setHomeData, homeData }) => {
-  const [squareFeet, setSquareFeet] = useState(homeData.squareFeet);
-  const [outdoor, setOutdoor] = useState(homeData.outdoor);
-  const [bedrooms, setBedrooms] = useState(homeData.bedrooms);
-  const [bathrooms, setBathrooms] = useState(homeData.bathrooms);
+const Page2: React.FC<Props> = ({ setHomeData, homeData }) => {
+  const [livableArea, setSquareFeet] = useState(String(homeData.livableArea));
+  const [outDoorArea, setOutdoor] = useState(String(homeData.outDoorArea));
+  const [bedroomCount, setBedrooms] = useState(String(homeData.bedroomCount));
+  const [bathroomCount, setBathrooms] = useState(
+    String(homeData.bathroomCount),
+  );
+
+  const navigation = useNavigation();
 
   const previousPage = () => {
-    setPage('page1');
+    navigation.goBack();
     setHomeData({
       ...homeData,
-      squareFeet,
-      outdoor,
-      bedrooms,
-      bathrooms,
+      livableArea: parseInt(livableArea, 10),
+      outDoorArea: parseInt(outDoorArea, 10),
```

Comment:
me being extremely nitpicky: could be called `outdoorArea` instead of `outDoorArea`. Feel free to reject this suggestion ;)

Reply:
Once again, this is correlated to the schema so I can't really change it.